src/validate.js

```diff
@@ -1,10 +1,33 @@
 import Ajv from 'ajv';
 import ajvKeywords from 'ajv-keywords';
 
-import absolutePathKeyword from './keywords/absolutePath';
+import addAbsolutePathKeyword from './keywords/absolutePath';
 
 import ValidationError from './ValidationError';
 
+/** @typedef {import("json-schema").JSONSchema4} JSONSchema4 */
```

Comment:
thats why I used in stylelint `stylelint` as package `import('stylelint').MyAwesomeType`. I think it does not matter much right now since there is no way to get `*.d.ts` files from jsdoc. Also I am curious about using reference here:
```
/// <reference path="../types/index.d.ts" />
```

Reply:
Now we can't use that, because our declarations generated from JSDoc